tplus/client/orderbook.py

```diff
@@ -323,6 +348,37 @@ async def get_user_orders_for_book(
         parsed_orders = parse_orders(response_data)
         return parsed_orders, response_data
 
+    @staticmethod
+    def compute_remaining(order: OrderResponse) -> int:
+        confirmed = int(order.confirmed_filled_quantity or 0)
+        pending = int(order.pending_filled_quantity or 0)
+        total_qty = int(order.quantity or 0)
+        return max(0, total_qty - confirmed - pending)
+
+    async def get_open_orders_for_book(
+        self, asset_id: AssetIdentifier, *, limit: int = 1000, max_pages: int = 50
+    ) -> list[OrderResponse]:
+        """Return only open, limit orders with remaining quantity > 0 for a book."""
+        # Pull pages until empty or cap reached
+        page = 0
+        all_orders: list[OrderResponse] = []
+        while page < max_pages:
+            parsed, raw = await self.get_user_orders_for_book(asset_id, page=page, limit=limit)
+            all_orders.extend(parsed)
+            if len(raw) < limit:
+                break
+            page += 1
+        open_orders: list[OrderResponse] = []
+        for o in all_orders:
+            if bool(o.canceled):
```

Comment:
can make all these conditions on one line

if bool(o. canceled) or o.limit_price is None ...